### PR TITLE
Padding-top added to heading in search page

### PR DESCRIPTION
**Pull request by alishamohanty**
This PR solves the issue in #224.

### CODE REVIEW

**Comment by alishamohanty** [comment]
@brenoferreira Could you review the PR regarding a small bug in #224.

**Comment by brenoferreira** [comment]
@alishamohanty Thank you for your contribution. Could you post any screenshots with the fix applied?

File: src/components/SearchGrid.vue [hunk 1]
```diff
@@ -168,6 +168,7 @@ export default {
 
   .search-grid_analytics h5,
   .search-grid_layout-control h5 {
+    padding-top: 13px;
```
Comment (brenoferreira) [inline src/components/SearchGrid.vue]: Could you use `vh` instead of fixed `px`?

Comment (ayushshivani) [reply src/components/SearchGrid.vue]: @brenoferreira Can I complete this task?

Comment (alishamohanty) [reply src/components/SearchGrid.vue]: I am just doing it now!

Comment (alishamohanty) [reply src/components/SearchGrid.vue]: Screenshot from 2019-03-27 02-16-48
Screenshot from 2019-03-27 02-11-16
Before and After photos!

**Comment by alishamohanty** [comment]
Made the required changes! Sorry for being late @brenoferreira for such minor bug!

File: src/components/SearchGrid.vue (hunk 1 above)
Comment (alishamohanty) [reply src/components/SearchGrid.vue]: Hey @brenoferreira could u have a look at this!

**Comment by brenoferreira** [comment]
@alishamohanty thank you for your contribution. This was fixed in #248

**Pull request closed without merging**